app/views/security_group/edit.html.haml

```diff
@@ -1,4 +1,9 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "securityGroupFormController"}
+%form#form_div{:name           => "angularForm",
+   "ng-controller" => "securityGroupFormController as vm",
+   "miq-form"      => true,
```

Comment:
Thanks.

I just realized that with the `"ng-cloak" => ''` setting you also need 
```ruby
"ng-show" => "vm.afterGet"
```
at this stage,  that way the form is presented to the user only after the REST API data is loaded in the models.

Sorry, I didn't catch that before.